deploy-service/teletraanservice/pom.xml

```diff
@@ -51,7 +51,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>6.0.23.Final</version>
+            <version>5.4.1.Final</version>
```

Comment:
Why reduce the version of the `hibernate-validator`?

Reply:
I used this version 5.4.1.Final at the very beginning and then github just bumped it for me here automatically: https://github.com/pinterest/teletraan/commit/8c0979c1e6277935cee6c6da337d5b34f4a017a1
After I approved and test locally, I found it could not launch the instance on my devapp. That's why I revert the version again here. ^^

Reply:
ok, let me do it